src/v3/src/components/Icon/OktaVerifyIcon.tsx

```diff
@@ -27,13 +27,6 @@ export const OktaVerifyIcon: FunctionComponent<IconProps> = ({
     viewBox="0 0 48 48"
   >
     <title id={name}>{description}</title>
-    <circle
-      cx="24"
-      cy="24"
-      r="24"
-      fill="#F5F5F6"
-      class="siwFillBg"
-    />
```

Comment:
@alexdahl-okta In your ticket you said "Remove background from Auth Coins throughout the widget, making them just Auth Icons." I want to make sure that is still true and we don't want ANY instances of our icons to have a background anymore? E.g. this `OktaVerifyIcon` is also rendered in the `LaunchAuthenticatorButton` and would lose its background as a result of this change
<img width="400" alt="Screenshot 2024-01-11 at 1 06 55 PM" src="https://github.com/okta/okta-signin-widget/assets/106110543/0b848f81-c850-4c85-9293-7dfdd4d851f8">

Reply:
Yes, thanks! Good for this context to also lose the circle background. 👍